packages/editor/src/components/post-title/style.native.scss

```diff
@@ -0,0 +1,7 @@
+
+.titleContainer {
+	padding-left: 16;
+	padding-right: 16;
+	padding-top: 32;
```

Comment:
This top padding looks a bit too much on the WordPress apps. I'm not sure if this was a requirement from before or it's an adjustment made for the example apps?
@iamthomasbishop might now :)

![screenshot_20190221-091118](https://user-images.githubusercontent.com/9772967/53175018-6f282780-35eb-11e9-9f0c-f0783a1e1533.png)

But not really important on this PR.

Reply:
We did add 16pt spacing above the title in a [related pr](https://github.com/wordpress-mobile/gutenberg-mobile/pull/616#issuecomment-465292533), is that what's at play here? Should we use margin instead of padding? I imagine when the block gets focused (with top/bottom borders) it'll looks odd.

Reply:
I see! Yes, adding the extra 16pts as margin will make the difference. Currently we don't see the top border when the title is selected.

Reply:
I just moved the style from one place to another, didn't change or add any styles(on the parentPR you can see that it was here before: https://github.com/wordpress-mobile/gutenberg-mobile/pull/622/files). This PR isn't changing anything about the paddings on the title. 

I've put some gifs here to demonstrate how it looks https://github.com/wordpress-mobile/gutenberg-mobile/pull/622#issuecomment-465984101